smoketest/scripts/cli/test_protocols_bfd.py

```diff
@@ -29,55 +29,83 @@
 vrf_name = 'red'
 peers = {
     '192.0.2.10' : {
-        'intv_rx'    : '500',
-        'intv_tx'    : '600',
-        'multihop'   : '',
-        'source_addr': '192.0.2.254',
-        'profile'    : 'foo-bar-baz',
-        'minimum_ttl': '20',
+        'intv_rx'           : '500',
+        'intv_tx'           : '600',
+        'multihop'          : '',
+        'source_addr'       : '192.0.2.254',
+        'profile'           : 'foo-bar-baz',
+        'minimum_ttl'       : '20',
```

Comment:
The original indents shouldn't be touched, as they were correct.
I still do not understand the real use case. 
Do any changes because if was configured in the Homelab looks strange to me.